### PR TITLE
Added MANIFEST.in file to include tests in the source distribution.

### PR DESCRIPTION
**Pull request by c0deaddict**
Looking at the flask repo (and reading [this](https://stackoverflow.com/a/36601576/248948) stackoverflow answer), adding a `MANIFEST.in` should solve #90.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,9 @@
+include CHANGELOG.md LICENSE README.md
+
+graft scripts
```
Comment (adamlwgriffiths) [inline MANIFEST.in]: I wouldn't include these, they're really for personal use so I don't have to remember commands.

File: MANIFEST.in [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+graft tests
+graft docs
```
Comment (adamlwgriffiths) [inline MANIFEST.in]: Should the rst be included?

**Comment by adamlwgriffiths** [comment]
Thanks for looking into this. I'd seen these before but was unaware of any difference between that and setup.py's functionality.
I've put some comments on the PR. Let me know what you think.

File: MANIFEST.in (hunk 2 above)
Comment (c0deaddict) [reply MANIFEST.in]: I guess you could see the raw docs as part of the package's source distribution. I've copied the MANIFEST.in file from [Flask](https://github.com/pallets/flask/blob/master/MANIFEST.in), they also have their raw docs in it.

Comment (adamlwgriffiths) [reply MANIFEST.in]: Ok, if they do it then let's follow suit.

**Comment by adamlwgriffiths** [comment]
Awesome, thanks for that.
I'll do a version bump and release it.

**Comment by c0deaddict** [comment]
Thanks you for reviewing!

**Pull request merged**